### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by abhishektanwar**
Files to check:

components/CardSlider/index.jsx
components/CardSlider/card-slider.css
components/ImageSlider/index.jsx
components/ImageSlider/image-slider.css
components/ProductCard/index.jsx
components/ProductCard/product-card.css
compoenents/screens/Home/index.jsx
compoenents/screens/Home/home.css

### CODE REVIEW

File: src/components/CardSlider/card-slider.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* css for div in which card slider is placed in home.jsx
+.card-slider-container{
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (srishtigautam21) [inline src/components/CardSlider/card-slider.css]: You can remove the commented code

File: src/components/CardSlider/index.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      <BadgeIconButton
+        badgeIconButtonWrapper="right-arrow"
+        icon={"fas fa-arrow-right"}
+        onClick={slideRight}
+      />
+      <div className="card-slider" ref={ref} >
+        {[1,2,3,4,5,6,7,8].map((slide,index)=>{
+          return (
+            // <div className="slider-card">
+
```
Comment (srishtigautam21) [inline src/components/CardSlider/index.jsx]: commented code

**Review by srishtigautam21 — APPROVED** [review]
The looks clean and understandable
Good to merge

File: src/components/Header/index.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <>
+      <Logo />
+      <SearchBar />
+      <div className="nav-section">
+        <BadgeIconButton
+          badgeIconButtonWrapper={"badge-icon-button-wrapper"}
+          badgeNumber={2}
+          icon={"far fa-heart"}
+          onClick={() => {
+            console.log("a");
```
Comment (zoef-yt) [inline src/components/Header/index.jsx:18]: Remove console log

File: src/components/ProductCard/index.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+import './product-card.css'
+const ProductCard = (props) => {
+  
+  const {imagePath,badgeText,title,price,actualPrice,discountPercentage} = props;
+  return (
+    // <div className='product-card-container'>
+    //   <div className="product-card-image">
+    //     <img src={imageUrl} alt="product-image" />
+    //   </div>
+    // </div>
```
Comment (zoef-yt) [inline src/components/ProductCard/index.jsx]: Remove commented code

File: src/screens/Home/index.jsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import Header from "../../components/Header";
+import ImageSlider from "../../components/ImageSlider";
+import { SliderData } from "../../components/ImageSlider/SliderData";
+import "./home.css";
+const Home = () => {
+  return (
+    <>
+      {/* <nav className="nav-bar shadow-box" id="my-nav-bar">
+        <Header />
+      </nav> */}
```
Comment (zoef-yt) [inline src/screens/Home/index.jsx]: Remove commented code

File: src/screens/Home/index.jsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import CardSlider from "../../components/CardSlider";
+// import FeaturedContainer from "../../components/FeaturedContainer";
```
Comment (zoef-yt) [inline src/screens/Home/index.jsx]: Remove commented code

File: src/screens/Home/index.jsx [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+import { SliderData } from "../../components/ImageSlider/SliderData";
+import "./home.css";
+const Home = () => {
+  return (
+    <>
+      {/* <nav className="nav-bar shadow-box" id="my-nav-bar">
+        <Header />
+      </nav> */}
+      <ImageSlider sliderData={SliderData} />
+      {/* <FeaturedContainer featuredContainerTitle={'Featured Categories'} /> */}
```
Comment (zoef-yt) [inline src/screens/Home/index.jsx]: Remove commented code

File: src/components/ImageSlider/index.jsx [hunk 8]
```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from "react";
```
Comment (zoef-yt) [inline src/components/ImageSlider/index.jsx:1]: Appreciation: The slider looks really great.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: src/components/CardSlider/card-slider.css (hunk 1 above)
Comment (abhishektanwar) [reply src/components/CardSlider/card-slider.css]: removed

File: src/components/CardSlider/index.jsx (hunk 2 above)
Comment (abhishektanwar) [reply src/components/CardSlider/index.jsx]: removed

File: src/components/Header/index.jsx (hunk 3 above)
Comment (abhishektanwar) [reply src/components/Header/index.jsx:18]: removed . fixed here https://github.com/abhishektanwar/VYBE-ECOMMERCE/pull/1#pullrequestreview-910464917

File: src/components/ProductCard/index.jsx (hunk 4 above)
Comment (abhishektanwar) [reply src/components/ProductCard/index.jsx]: removed

File: src/screens/Home/index.jsx (hunk 5 above)
Comment (abhishektanwar) [reply src/screens/Home/index.jsx]: removed.

File: src/screens/Home/index.jsx (hunk 6 above)
Comment (abhishektanwar) [reply src/screens/Home/index.jsx]: removed

File: src/screens/Home/index.jsx (hunk 7 above)
Comment (abhishektanwar) [reply src/screens/Home/index.jsx]: removed

**Review by zoef-yt — APPROVED** [review]
Good to merge. Good work.

**Pull request merged**